demo/index.html

```diff
@@ -29,17 +29,26 @@
     <h1>&lt;iron-media-query&gt;</h1>
 
     <template is="dom-bind">
-      <iron-media-query query="(min-width: 600px)" query-matches="{{queryMatches}}"></iron-media-query>
+      <iron-media-query query="(min-width: 600px)" query-matches="{{wide}}"></iron-media-query>
+      <iron-media-query full query="print" query-matches="{{print}}"></iron-media-query>
 
-      <template is="dom-if" if="{{queryMatches}}">
+      <template is="dom-if" if="{{wide}}">
```

Comment:
so happy about the `s/queryMatches/wide`. It was confusing people before :beers: